src/styles/_catalog.scss

```
@@ -1,22 +1,32 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@mixin query ($cols) {
+  grid-template-columns: repeat($cols, 200px);
+}
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-content: start;
+  gap: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 1024px) {
+    @include query(4);
+  }
+
+  @media (max-width: 1024px) and (min-width: 768px) {
+    @include query(3);
+  }
+
+  @media (max-width: 768px) and (min-width: 488px) {
+    @include query(2);
+  }
+
+  @media (max-width: 488px) {
+    @include query(1);
```

```suggestion
  @media (min-width: 1024px) {
    @include query(4);
  }

  @media (min-width: 768px) {
    @include query(3);
  }

  @media (min-width: 488px) {
    @include query(2);
```
and add `grid-template-columns: 1, 200px` to the `catalog` as a default value